### DISCUSSION TITLE
Torus showWallet requires to login again

### DISCUSSION THREAD

**Discussion by willydouhard** [Web3Auth/Web3Auth#533] [Help]
I am using the torus plugin in my react application.

Here are the dependencies I use:
```
    "@web3auth/base": "^2.0.0",
    "@web3auth/torus-wallet-connector-plugin": "^2.0.0",
    "@web3auth/web3auth": "^2.0.2",
    "ethers": "^5.6.8",
    "react": "^17.0.2",
```
The login part works, however when I call `torusPlugin.torusWalletInstance.showWallet("home")`, a pop up opens but requires the user to login again:
<img width="1312" alt="Capture d’écran 2022-09-01 à 18 40 22" src="https://user-images.githubusercontent.com/13104895/187967536-67748dc7-6679-4ca5-849d-d6c982b7d7ef.png">

I am pretty sure the user is actually logged in (in my react application) since I get the correct `userInfos`:

```
aggregateVerifier: "tkey-google"
dappShare: ""
email: "xxxw@gmail.com"
idToken: "xxx"
name: "xxx"
oAuthAccessToken: ""
oAuthIdToken: ""
profileImage: "https://lh3.googleusercontent.com/a/AItbvmk7Ko7oVrDOCIB6HvREwOvN-F4rszSyfkgTy2I3=s96-c"
typeOfLogin: "google"
verifier: "torus"
verifierId: "xxx@gmail.com"
```

A couple days ago, this was working correctly. Any idea?

**Comment by willydouhard** [comment]
It is now working, I have no idea why though.

**Reply by himanshuchawla009** [reply]
It was an intermittent issue in torus wallet, we have fixed it yesterday. it should be working now.

**Comment by himanshuchawla009** [comment]
It was an intermittent issue in torus wallet, we have fixed it yesterday. it should be working now.
